src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -297,12 +297,8 @@ MulticopterAttitudeControl::Run()
 
 		bool attitude_setpoint_generated = false;
 
-		const bool is_hovering = (_vehicle_type_rotary_wing && !_vtol_in_transition_mode);
-
-		// vehicle is a tailsitter in transition mode
-		const bool is_tailsitter_transition = (_vtol_tailsitter && _vtol_in_transition_mode);
```

Comment:
Can you entirely remove `_vtol_tailsitter` now?

Reply:
outdated